packages/shared/assetdb.ts

```diff
@@ -117,6 +117,20 @@ export async function readAsset({
   return { asset, metadata };
 }
 
+/**
+ * Deletes the passed in asset if it exists and ignores any errors
+ * @param userId the id of the user the asset belongs to
+ * @param assetId the id of the asset to delete
+ */
+export async function silentDeleteAsset(
+  userId: string,
+  assetId: string | undefined,
```

Comment:
Tbh, not convinced that an API in `assetdb` that's called `silentDeleteAsset` accepts an optional assetId. If we really care about this, I'd move it as a helper function in crawlerWorker. Not sure it's worth it.